addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/PresenceDetection.java

```diff
@@ -186,11 +186,29 @@ public void setUseArpPing(boolean enable, String arpPingUtilPath, @Nullable Inet
             arpPingMethod = ArpPingUtilEnum.UNKNOWN_TOOL;
             return;
         } else if (destinationAddress == null || !(destinationAddress instanceof Inet4Address)) {
-            arpPingState = "Destination is not IPv4";
+            arpPingState = "Destination is not a valid IPv4 address";
             arpPingMethod = ArpPingUtilEnum.UNKNOWN_TOOL;
             return;
         }
         arpPingMethod = networkUtils.determineNativeARPpingMethod(arpPingUtilPath);
+        switch (arpPingMethod) {
+            case UNKNOWN_TOOL: {
+                arpPingState = "Unknown arping tool";
```

Comment:
You could also store these strings in the enum, then they are closer together, or maybe using a map. This could also work but imho the other options are a little bit more elegant.

Reply:
I might want to apply I18n at some point and thus not hardcoding the strings into the Enum. I do not expect that enum to change a lot, and if it falls through, there is just no name exposed to the user.